app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -482,4 +482,16 @@ public static void reportNotificationType(String notificationType) {
         reportEvent(CCAnalyticsEvent.CCC_NOTIFICATION_TYPE,
                 CCAnalyticsParam.NOTIFICATION_TYPE, notificationType);
     }
+
+    public static void reportHQInvitationDeepLink(String hqDomain) {
+        reportEvent(CCAnalyticsEvent.HQ_INVITE_LINK,
+                CCAnalyticsParam.CCHQ_DOMAIN, hqDomain);
+    }
+
+    public static void reportHQInvitationResponse(String hqDomain, boolean accepted, String method) {
```

Comment:
I see that we are sending errors for the `method` parameter. Is that intentional ? If so we should rename the parameter as `reason` to signify the fail reason.